docs/UserGuide.md

```diff
@@ -225,6 +388,16 @@ Format: `done <task_index>`
 Example of usage: 
 * `done 1`
 
+Example of output:
+```
+Command: 
+⋗	done 1
+════════════════════════════════════════════════════════════════════════════════
+The task has been successfully marked as complete.
+
+════════════════════════════════════════════════════════════════════════════════
+```
+
```

Comment:
Maybe show the output of
1. `list -t`
2. `done 1`
3. `list -t`
Or it would be wise to show the task instead of `The task has been successfully marked as complete.`